src/configs/recommended.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     'primer-react/new-color-css-vars': 'error',
     'primer-react/a11y-explicit-heading': 'error',
     'primer-react/no-deprecated-props': 'warn',
+    'primer-react/a11y-link-in-text-block': 'error',
```

Comment:
Given that `inline` is specific to dotcom's setup, maybe this should not be turned on by default. would love to hear what others think!

Reply:
True, we shouldn't turn it on by default